deploy/manifests/base/kong-ingress-dbless.yaml

```diff
@@ -102,7 +108,7 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
-        image: kong/kubernetes-ingress-controller:1.3
+        image: kong/kubernetes-ingress-controller:2.0
```

Comment:
should we follow semver ? i.e 1.3.x, 2.0.x

Reply:
Each of our releases follows a `major.minor.patch` scheme.
We also maintain `major.minor` tags that point to the latest patch release, as a convenience measure.

Our intention is that, if we release a new patch version with bugfixes, we don't need to update the manifests and the helm chart.
But, since there's `imagePullPolicy: IfNotPresent` below, users' clusters where the image is cached already won't pick up the new version. I'll create an issue for that.